cluster_traffic_capture/docker_config/mirror.conf

```diff
@@ -0,0 +1,12 @@
+[mirror]
```

Comment:
To Mikayla's point about preferring jinja...  There is a weird impedance mismatch w/ the mirror conf file being defined here and the general one being entirely within strings of the python code.